src/main/javatests/com/google/summit/translation/ModifierTest.kt

```diff
@@ -168,4 +171,31 @@ class ModifierTest {
 
     assertThat(annotationE.args).isEmpty()
   }
+
+  @Test
+  fun anonymousInitialization_hasCorrectModifiers() {
+    val input =
+      """
+        class Test {
+          {
+            print('init');
+          }
+          static {
+            print('more init');
+          }
+        }
+        """
+    val classDecl = TranslateHelpers.parseAndFindFirstNodeOfType<ClassDeclaration>(input)
+
+    assertNotNull(classDecl)
+    assertThat(classDecl.methodDeclarations).hasSize(2)
+
+    val normalInitializer = classDecl.methodDeclarations.first()
+    assertThat(normalInitializer.modifiers).isEmpty()
+
+    val staticInitializer = classDecl.methodDeclarations.last()
+    assertThat(staticInitializer.modifiers).hasSize(1)
+    assertThat((staticInitializer.modifiers.first() as? KeywordModifier)?.keyword)
```

Comment:
This is surprisingly verbose for such a common check. Any suggestions here, to extend the summit API to make it more natural to query? Perhaps only exposing the raw list of (annotation + keyword) modifiers isn't the right thing...

Reply:
In practice, this would be more convenient than it appears here through the use of `find`, `any`, etc.

I agree, though, which is why I locally added a simple additional function `hasKeyword` in `Declaration` that has already seen a lot of use.

We could split up `modifiers` into separate `annotationModifiers` and `keywordModifiers` properties like `ClassDeclaration` does to its `bodyDeclarations`. Or, to maintain the current structure, we can add them as getters instead. Either of these two would already help in many situations.

I'm sure I will find more places to improve the interface while interacting with Summit, but this would be a good start. What do you think?

Reply:
Agreed, `hasKeyword` seems like a great addition.

Both of your proposal for `modifiers` seem fine to me. Since both the allowable keywords and annotations are small and fixed by the Apex language, these are all small constant-time operations even for antagonistic inputs. Feel free to go with whichever looks better to you and/or is faster to implement.